src/validation/tests/duplicates_validation_test.rs

```diff
@@ -0,0 +1,202 @@
+use crate::{ast::SourceRange, diagnostics::Diagnostic, test_utils::tests::parse_and_validate};
```

Comment:
Some more examples:
A global variable and a program with the same name
A global function block instance and a function of the same name